Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -245,31 +63,44 @@ Write-Output "Load generator machine type = $machineType"
 Write-Output "Run source identifier = build/$env:SYSTEM_DEFINITIONID/$env:BUILD_BUILDID"
 
 #Validate Input
-ValidateInputs
+ValidateInputs $websiteUrl $env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI $connectedServiceName
 
-$connectedServiceDetails = Get-ServiceEndpoint -Context $distributedTaskContext -Name $connectedServiceName
+if([string]::IsNullOrWhiteSpace($connectedServiceName))
+{
+	$connectedServiceDetails = Get-ServiceEndpoint -Context $distributedTaskContext -Name SystemVssConnection
+}
+else
+{
+    $connectedServiceDetails = Get-ServiceEndpoint -Context $distributedTaskContext -Name $connectedServiceName
+}
```

Comment:
I wanted to keep the ValidateInputs as a part of the CltTasksUtility so that it could be reused across tasks.
However, for security reasons Powershell does not allow passing the OAuth token to a different module as it is a secure string object.